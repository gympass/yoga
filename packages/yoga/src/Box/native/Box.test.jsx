import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, Box, Text } from '../..';

describe('<Box />', () => {
  it('should match snapshot', () => {
    const { toJSON } = render(
      <ThemeProvider>
        <Box
          b="small"
          p="small"
          borderRadius="small"
          color="feedback.success.dark"
        >
          <Text>Making wellbeing universal</Text>
        </Box>
      </ThemeProvider>,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
