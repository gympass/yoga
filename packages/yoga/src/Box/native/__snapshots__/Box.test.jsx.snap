// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> should match snapshot 1`] = `
<View
  b="small"
  borderRadius="small"
  color="feedback.success.dark"
  p="small"
  style={
    Array [
      Object {
        "borderColor": "black",
        "borderRadius": 8,
        "borderStyle": "solid",
        "borderWidth": 1,
        "color": "#1D856C",
        "paddingBottom": 16,
        "paddingLeft": 16,
        "paddingRight": 16,
        "paddingTop": 16,
      },
    ]
  }
>
  <Text
    inverted={false}
    light={false}
    size="medium"
    style={
      Array [
        Object {
          "color": "#231B22",
          "fontFamily": "Rubik",
          "fontSize": 16,
          "fontWeight": "400",
          "lineHeight": 24,
        },
      ]
    }
  >
    Making wellbeing universal
  </Text>
</View>
`;
